lib/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -177,6 +177,8 @@ a {
 
 ### `ignoreLonghands: ["array", "of", "properties"]`
 
+Ignore the specified longhand properties when checking for redundant longhands.
```

Comment:
```suggestion
Ignore the specified longhand properties.
```

Let's remove the redundancy as we favour keeping our docs concise because we have so many of them.